src/stylesheets/reusables/form.scss

```diff
@@ -22,6 +22,12 @@ form {
     }
   }
 
+  .alert.alert-danger {
```

Comment:
Can you make this a child instead?

Reply:
What do you mean--make it a child of what?

Reply:
````
  .alert {
    &.alert-danger { 
     .....
```